### PR TITLE
Simplify memoization to avoid refresh control jumping

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

File: src/components/asset-list/RecyclerAssetList/index.tsx [hunk 1]
```diff
@@ -691,7 +692,7 @@ function RecyclerAssetList({
     lastIsCoinListEdited,
     lastOpenFamilyTabs,
     lastSections,
-    sections,
+    //sections,
```
Comment (brunobar79) [inline src/components/asset-list/RecyclerAssetList/index.tsx:695]: Remove?

**Review by brunobar79 — APPROVED** [review]

**Comment by osdnk** [comment]
@skylarbarrera feel free to merge if you wanna test it

**Comment by skylarbarrera** [comment]
@osdnk getting the following on android when ever you open smallBalances http://cloud.skylarbarrera.com/Screen-Recording-2021-03-22-15-59-13.mp4

**Comment by osdnk** [comment]
@skylarbarrera can you try now?

**Review by skylarbarrera — APPROVED** [review]
lgtm, some unrelated android issues im gonna ping you about

**Pull request merged**